lib/checkPullRequestLabels.js

```diff
@@ -285,6 +285,13 @@ module.exports.checkChangelogLabel = async function(context) {
       'guidance. Thanks!',
   });
   await context.github.issues.createComment(stateCommentParams);
+
+  // Assign the author.
```

Comment:
The issue here is what if the author does not have assigning privileges? In that case we should assign/ping the requested reviewers to add the changelog label.

Reply:
We are already adding the default changelog label for non-collaborators, hence this will only run for collaborators.

Reply:
Ah, right. I missed that - can you please update the comment to explain the same?

Reply:
Done.